.github/workflows/release.yaml

```diff
@@ -20,11 +24,9 @@ jobs:
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
 
       - name: Install Helm
-        uses: azure/setup-helm@v1
-        with:
-          version: v3.4.0
+        uses: azure/setup-helm@v3
 
       - name: Run chart-releaser
-        uses: helm/chart-releaser-action@v1.2.1
+        uses: helm/chart-releaser-action@v1.5.0
```

Comment:
A `chart-releaser-action@v1.5.0` requires to update syntax on `.github/workflows/chart-testing.yaml` line 36

```
------
.github/workflows/chart-testing.yaml:36:14: workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions [deprecated-commands]
   |
36 |         run: |
   |              ^
------
```